examples/injections/functions.yaml

```diff
@@ -17,15 +17,15 @@ injections: |-
       - name: header.status
         widget: Badge
         highlights:
-          success:
+          positive:
             - RUNNING
-          error:
+          negative:
```

Comment:
shouldn't this one be called "warning" for consistency with the documentation and the other files?